containers/java-postgres/README.md

```diff
@@ -0,0 +1,132 @@
+# Java & PostgreSQL
```

Comment:
Let's reorder the content of the file to get the reader started quickly.

It's probably most common that a reader is looking for how to start the containers and test that they are running properly. Let's move that content up.

Customization content can come after that. Please take another pass at the contaent abotu customization option and add some structure with headers and/or bulleted lists. Examples when options are needed would make a nice addition.

Reply:
hi @xtophs, I think this `java-postgres` readme does follow the suggested [container-template/docker-compose](https://github.com/microsoft/vscode-dev-containers/tree/main/container-templates/docker-compose) version/format, even though ours was also structured after the `java` and `python-3-postgres` ones

Reply:
What if we keep the order and add a table of contents? The structure should match the other files to prevent confusion, but by having a ToC we'll be able to help a reader jump to whatever they think is most important.